index.js

```diff
@@ -19,6 +22,26 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    // add a new recipe
+    const bolonhese = ({title: 'Spaghetti Bolonhese', level: 'Easy Peasy', cuisine: 'italian', });
+    await Recipe.create(bolonhese);
+    console.log(bolonhese.title);
+
+    // insert multiple recipes
+    await Recipe.insertMany(data);
+    data.forEach(recipes => console.log(recipes.title));
```

Comment:
Here you are console logging all the recipe's title from the `data.json` file. But the purpose of this step was to console.log it through the `database`:

```
    const manyRecipes = await Recipe.insertMany(data);
    manyRecipes.forEach((recipe) =>
      console.log(`New recipe added: ${recipe.title}`)
    );
```